platform/authorized_api/access_control/public_v2.yaml

```diff
@@ -1,4 +1,5 @@
 openapi: 3.0.3
+x-api-category: Access Control
```

Comment:
That file update is not necessary since it only brings `x-api` tags that will be used with new Developer Portal, so it's safe to skip it entirely.

Reply:
and btw i see sth is wrong [here](https://docs.cldsrv.biz/master/promil/development/apis/http?spec=apps#operation/api_apps_details_v2), i see app details three times

Reply:
and what about [that](https://github.com/PiwikPRO/PPMS-PublicDocs/pull/1182#discussion_r1782489895)?